canonical-web-and-design/ubuntu.com
# Add French migrate from vmware to openstack whitepaper

## Done

- Add French migrate from vmware to openstack whitepaper
    - takeover
    - takeover index
    - whitepaper engage page
	- engage index
- Moved language based takeovers into lang code based folders and updated links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and https://0.0.0.0:8001/engage/fr/migrer-de-vmware-a-openstack - don't forget to change your chrome://settings/languages to french
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [brief](https://docs.google.com/spreadsheets/d/1OB2ERfClOQc8KV64BT66GaAQ1QQJ-4RIkwTmk83l314/edit#gid=1271849439) and [copy doc](https://docs.google.com/document/d/1yrfVZnXhewJ1q2YZoJyJtIEdXXtSeKHNFqGNnuIYiGc/edit)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2456

## Screenshots

![image](https://user-images.githubusercontent.com/441217/77848075-05992600-71ba-11ea-8b50-e1ffca6e3245.png)
